5-appinfra/modules/cicd-pipeline/variables.tf

```
@@ -29,9 +29,14 @@ variable "env_cluster_membership_ids" {
   }))
 }
 
-variable "service" {
+variable "service_name" {
   type        = string
-  description = "service name (e.g. 'frontend')"
+  description = "service name (e.g. 'transactionhistory')"
+}
+
+variable "team_name" {
+  type        = string
+  description = "team name (e.g. 'ledger')"
```

This variable is now only being used to prefix the CI Build Trigger;

```suggestion
  description = "Team name (e.g. 'ledger'). This will be the prefix to the service CI Build Trigger Name."
```